### DISCUSSION TITLE
Get Document Type

### DISCUSSION THREAD

**Discussion by sberer** [regulaforensics/DocumentReader-web-csharp-client#29] [Q&A]
How can I get the document type such as driving license, Id-Card or passport?

**Comment by hleb-albau** [comment]
There is `Result.DOCUMENT_TYPE` result, which containing `doc_type_result.one_candidate.fdsid_list.d_type` field. This field contains document type value. See https://github.com/regulaforensics/DocumentReader-web-openapi/blob/master/e-document-type.yml

**Comment by sberer** [comment]
I did it this way:
`ChosenDocumentTypeResult chosenDocumentTypeResult = response.ResultByType<ChosenDocumentTypeResult>(Result.DOCUMENT_TYPE);
            var documentType = chosenDocumentTypeResult?.OneCandidate?.FDSIDList?.DType;`

Is this correct?

**Reply by hleb-albau** [reply]
Yes, it is correct. 

Note, you have to pass `Result.DOCUMENT_TYPE` in a request if you specify concrete results (by default process method returns all possible results for an image). See https://github.com/regulaforensics/DocumentReader-web-csharp-client/blob/9780111b69c916056803a05fa0fbbd05de263fb0/src/Regula.DocumentReader.NetCoreExample/Program.cs#L35
